package/gluon-mesh-babel/src/Makefile

```diff
@@ -24,7 +24,7 @@ CFLAGS_JSONC = $(shell pkg-config --cflags json-c)
 LDFLAGS_JSONC = $(shell pkg-config --libs json-c)
 
 
-respondd.so: respondd.c handle_neighbour.c
```

Comment:
the `.c` file could now be removed and added as-is to `neighbours-babel.c`

Reply:
agreed. done.